lib/msf/core/payload/windows/reverse_http.rb

```diff
@@ -350,13 +360,33 @@ def asm_reverse_http(opts={})
         call ebp
         test eax,eax
         jnz allocate_memory
+      ^
+
+    if retry_wait
```

Comment:
Rather that supporting a lack of a wait, why not have a sane default, such as a few seconds?